src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -29,12 +29,13 @@ class FilesystemLoader extends \Twig_Loader_Filesystem
     /**
      * Constructor.
      *
-     * @param FileLocatorInterface        $locator A FileLocatorInterface instance
-     * @param TemplateNameParserInterface $parser  A TemplateNameParserInterface instance
+     * @param FileLocatorInterface        $locator  A FileLocatorInterface instance
+     * @param TemplateNameParserInterface $parser   A TemplateNameParserInterface instance
+     * @param string                      $rootPath The project's root dir
      */
-    public function __construct(FileLocatorInterface $locator, TemplateNameParserInterface $parser)
+    public function __construct(FileLocatorInterface $locator, TemplateNameParserInterface $parser, $rootPath = '')
```

Comment:
My aim was to avoid the `getcwd` because this is the issue in a Symfony context (https://github.com/twigphp/Twig/issues/2281).
But OK, the job is done by the compiler pass.